src/proto/api_messages-test.js

```diff
@@ -184,6 +184,7 @@ describe('AnalyticsContext', () => {
     timestamp.setSeconds(0);
     timestamp.setNanos(0);
     analyticscontext.setClientTimestamp(timestamp);
+    analyticscontext.setReaderSurfaceType(ReaderSurfaceType.READER_SURFACE_TYPE_UNSPECIFIED);
```

Comment:
It looks like this indentation is off. Is this how it was written by the script?

Reply:
Nevermind, I guess it was just confusing presentation on Github.